core/templates/dev/head/pages/exploration_player/HintButtonDirective.js

```diff
@@ -26,7 +26,8 @@ oppia.directive('hintButton', [
       scope: {
         onClickHintButton: '&',
         allHintsAreExhausted: '&',
-        currentHintIsAvailable: '&'
+        currentHintIsAvailable: '&',
+        isSupplementalCard: '='
```

Comment:
Should use '&', since you don't need the two-way binding? Similar to allHintsAreExhausted.

OK to define new variable in supplemental card directive, if necessary.

Reply:
I frankly have no idea what was going on with this, whenever I used any binding other than '=', and I logged out $scope.isSupplementalCard, it would log what I was expecting, but whenever I used it in a boolean expression it didn't seem to be evaluated properly... as in 

```
$scope.getTooltipPlacement = function() {
            return (viewportIsNarrow && $scope.isSupplementalCard) ?
              'right' : 'left';
          };
```

evaluates to ` 'right' ` even though ` $scope.isSupplementalCard ` is ` false `.